apps/z2m_ikea_controller/z2m_ikea_controller.py

```diff
@@ -320,6 +322,8 @@ def change_light_state(self, old, attribute, direction, steps):
         min_ = self.attribute_minmax[attribute]["min"]
         step = (max_ - min_) // steps
         new_state_attribute = old + sign * step
+        if self.from_off_to_min_brightness and attribute == "brightness" and  self.get_state(self.light["name"])=="off":
```

Comment:
```suggestion
if self.from_off_to_min_brightness and attribute == ATTRIBUTE_BRIGHTNESS and  self.get_state(self.light["name"])=="off":
```
Also, I would put this logic on the `click` function since this function is called as well for `hold` as well. Or should the light also turn on when the brightness up is hold?

Reply:
My mistake, I didn't notice the `ATTRIBUTE_BRIGHTNESS` global.

Reply:
Yeah I think that also just clicking once the brightness up button on a controller would turn the lamp on at minimum brightness, but I'm not sure. Tomorrow I'll perform some tests with an `E1810` and a random IKEA bulb then I'll report the results back here.